diff --git a/src/backend/pyproject.toml b/src/backend/pyproject.toml
index 509e75a3..ec432b99 100644
--- a/src/backend/pyproject.toml
+++ b/src/backend/pyproject.toml
@@ -17,13 +17,13 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.11",
 ]
 description = "An application to print markdown to pdf from a set of managed templates."
 keywords = ["Django", "Contacts", "Templates", "RBAC"]
 license = { file = "LICENSE" }
 readme = "README.md"
-requires-python = ">=3.12"
+requires-python = ">=3.11"
 dependencies = [
     "beautifulsoup4==4.13.4",
     "boto3==1.38.27",
